src/main/scala/org/broadinstitute/dsde/agora/server/model/AgoraEntityModel.scala

```diff
@@ -0,0 +1,9 @@
+
+package org.broadinstitute.dsde.agora.server.model
+
+import java.util.Date
+
+case class AgoraMetadata(id: Option[Integer] = None, namespace: String, name: String, synopsis: String,
```

Comment:
I get why you ordered this the way you did but putting defaulted params before non-defaulted params can lead to pain & anguish